src/main/java/com/epam/productsreview/ProductsReviewApplication.java

```diff
@@ -0,0 +1,13 @@
+package com.epam.productsreview;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+@SpringBootApplication
+public class ProductsReviewApplication {
```

Comment:
This service only deals with products(It will in turn call review service later). Please change app name and package name like ProductApplication

Reply:
modified